wiki/fr/fr-Choosing-a-Server-Type.md

```diff
@@ -40,18 +42,17 @@ Ce mode est activé en décochant la case "Rendre mon serveur public...".
 
 Lorsqu'il est exécuté sans l'interface graphique ("headless"), ce qui suit configurerait un serveur privé :
 
-```shell
-Jamulus --nogui --server
-```
+```shell Jamulus --nogui --server ```
 
 Voir également [Options de la ligne de commande](Command-Line-Options) pour les autres paramètres que vous pouvez définir.
 
 ### 3. Serveur annuaire
+
 La configuration de votre serveur en tant que serveur annuaire ne doit être effectuée que dans des circonstances particulières (par exemple, pour des événements en ligne ou des associations musicales). La plupart des gens peuvent ignorer ce type.
 
-Pour afficher les serveurs répertoriés par un serveur d'annuaire personnalisé, les musiciens doivent saisir l'adresse dans le champ de paramètres "Adresse personnalisée du serveur annuaire" de leur client.  Ils verront alors une liste de configuration de connexion générée par ce serveur d'annuaire.
+Pour afficher les serveurs répertoriés par un serveur d'annuaire personnalisé, les musiciens doivent saisir l'adresse dans le champ de paramètres "Adresse personnalisée du serveur annuaire" de leur client. Ils verront alors une liste de configuration de connexion générée par ce serveur d'annuaire.
 
-Les opérateurs de serveurs normaux peuvent également s'inscrire auprès de votre serveur d'annuaire personnalisé afin que leurs serveurs apparaissent dans votre liste de serveurs en définissant le votre comme option `--centralserver`.
+Les opérateurs de serveurs normaux peuvent également s'inscrire auprès de votre serveur d'annuaire personnalisé afin que leurs serveurs apparaissent dans votre liste de serveurs en définissant le votre comme option `--centralserver`. i
```

Comment:
What's the i for?

Reply:
Oops, must have slipped in due to a careless finger...